### DISCUSSION TITLE
Enterobacterales variable function

### DISCUSSION THREAD

**Discussion by theanita1** [msberends/AMR#167] [Ideas]
Could it be possible to have a function which creates a binary variable (could be a logical) that scrapes the organism code variable and says "yes" or "no" if the organism is an Enterobacterales? 

Here's my coding I've used so far but if it could be nicely wrapped up into a mo_enterobacterales() or something similar? 
```
df$order<- mo_order(df$org_code)
df<- df %>% mutate(enterobact = case_when(order == "Enterobacterales" ~ "Yes",
                                 T ~ "No"))
df<- subset(df, select = -c(order))
```

**Comment by msberends** [comment]
Hi,

I can surely see the point in that, but also think it is too specific. We could add tens or hundreds of functions so support all kinds of filters on specific taxonomic properties. I think we shouldn’t. 

You can however make your code much easier and faster:

```r
df <- df %>%
  mutate(enterobact = mo_order(org_code) == "Enterobacterales")
```

Or directly in an `if()`:

```r
if (mo_order(df$org_code) == "Enterobacterales") {
  …
}

if (mo_order(df$org_code) == "Enterobacterales") {
  …
} else {
  …
}
```

**Reply by theanita1** [reply]
Fair, no worries and thanks for the coding advice!
